websites/A/AniMixPlay/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Jack Hansen",
+    "id": "140590672657776641"
+  },
+  "service": "AniMixPlay",
+  "description": {
+    "en": "AniMixPlay is a website that allows you to watch Anime from all around the world, completely free."
+  },
+  "url": "animixplay.to",
+  "version": "1.0.1",
+  "logo": "https://i.imgur.com/eMXyO0Y.png",
```

Comment:
maybe more quality logo since this one has white bar at left

Reply:
I can't see a white bar on this logo, could you send a screenshot please? :0

Reply:
![image](https://user-images.githubusercontent.com/34792838/102093121-15789900-3e21-11eb-9734-24d2bddb1d87.png)
Looking good to me :/

Reply:
#BlameGoogleDown
![b4QzjyXF](https://user-images.githubusercontent.com/31831814/102093792-7ac88c00-3e19-11eb-871c-035c731ef565.png)

Reply:
![Screen Shot 2020-12-14 at 15 34 53](https://user-images.githubusercontent.com/9348108/102093857-8050b280-3dd6-11eb-9a09-b6b6a26f5266.png)
for me it looks like this